src/documentation/runners/apex.md

```diff
@@ -5,5 +5,108 @@ permalink: /documentation/runners/apex/
+
+#### 1. Steps to setup your Sandbox
+If you have your own Hadoop setup then you can skip this section.
+
+* Download the [DataTorrent RTS Sandbox](http://google.com/)
+* [Download](https://www.virtualbox.org/wiki/Downloads) and [install](https://www.virtualbox.org/manual/ch02.html) Virtualbox on your machine
+* Import Sandbox in virtualbox and start the VM
+* Update YARN setting as follows:
+	Update following property in yarn-site.xml at  ~/hadoop/etc/hadoop/yarn-site.xml
+```
+  <property>
+    <name>yarn.scheduler.minimum-allocation-mb</name>
+    <value>8</value>
+  </property>
+```
+
+* Restart YARN
+```
+hadoop/sbin/stop-yarn.sh && hadoop/sbin/start-yarn.sh
+```
+
+#### 2. Download code and compile
+* Install git
+```
+sudo apt-get install -y git
+```
+
+* Beam code
+```
+mkdir code && cd code
+git clone https://github.com/apache/incubator-beam.git
+pushd incubator-beam
+mvn clean install -DskipTests
+popd
+```
+
+* Wordcount example to run on Apex-Runner
+```
+git clone https://github.com/tweise/apex-samples.git
+pushd apex-samples/beam-apex-wordcount
+mvn clean install -DskipTests
+```
+
+## Running wordcount using Apex-Runner
+Currently the wordcount program reads data from /tmp/input/ and stores results into /tmp/output/ on HDFS.
+
+* Download some data for processing and put it on HDFS
+```
+curl http://www.gutenberg.org/cache/epub/1128/pg1128.txt > /tmp/kinglear.txt
+hdfs dfs -mkdir -p /tmp/input/
+hdfs dfs -put /tmp/kinglear.txt /tmp/input/
+```
+
+* The output directory should not exist on HDFS. Delete it if it exists.
+```
+hdfs dfs -rm -r -f /tmp/output/
+```
+
+* Run the wordcount example
+```
+apex
+```
+
+* You will get apex prompt. On the apex prompt, run following command.
+```
+launch ~/code/apex-samples/beam-apex-wordcount/target/beam-apex-wordcount-1.0-SNAPSHOT.apa
+```
+
+This will launch an Apex application. To exit from apex prompt, use “exit” or Control-d.
+
+## Checking output
+
+The sample program which is processing small amount of data would finish quickly. You can check contents on /tmp/output/ on HDFS
+```
+hdfs dfs -ls /tmp/output/
+```
+
+You should see couple of files in the output
+```
+/tmp/output/_SUCCESS
+/tmp/output/part-r-00000
+```
+
+The presence of `_SUCCESS` file in the output folder signifies successful completion of the program. The output of wordcount program is in `part-r-00000` file. You can check output of the program as
+```
+hdfs dfs -cat /tmp/output/part-r-00000
+```
+
+## Montoring progress of your job
+
+If you are using DataTorrent RTS Sandbox then you can monitor progress of the application on http://localhost:9090/ using your browser. The console provides you updates on the application progress, general statistics like number of operators, number of containers - planned and allcated, allocated memory to the application etc.
```

Comment:
Depending on your installation, you may be able to monitor the progress of your job on the Hadoop cluster. Please see the relevant information on the Apache Apex website. (add specific links)